src/main/resources/edu/cornell/kfs/tax/batch/DefaultTransactionRowOutputDefinition.xml

```diff
@@ -81,7 +85,11 @@
         <field name="Field25" length="14" type="DETAIL" value="foreignSourceIncomeIndicator" />
         <field name="Field26" length="19" type="DETAIL" value="federalIncomeTaxPercent" />
         <field name="Field27" length="100" type="DETAIL" value="paymentDescription" />
-        <field name="Field28" length="45" type="DETAIL" value="paymentLine1Address" />
+        <field name="Field28" length="55" type="DETAIL" value="paymentLine1Address" />
+        <field name="Field28" length="55" type="DETAIL" value="paymentLine2Address" />

```

Comment:
There are multiple fields named "Field28", I would suggest using unique names to avoid confusion.